src/atomic.rs

```diff
@@ -801,6 +801,8 @@ impl<'scope, T> Clone for Ptr<'scope, T> {
     }
 }
 
+impl<'scope, T> Copy for Ptr<'scope, T> {}
```

Comment:
Oh, I think I know what's up. Derived `Clone` and `Copy` impls require `T: Clone`.
The error occurs because there is no implementation of `Copy` since `Entry: !Clone`.